tests/test_synology_dsm.py

```diff
@@ -904,7 +904,10 @@ def test_surveillance_station(self):
         assert self.api.surveillance_station.get_all_cameras()
         assert self.api.surveillance_station.get_camera(1)
         assert self.api.surveillance_station.get_camera_live_view_path(1)
-        assert self.api.surveillance_station.get_camera_live_view_path(1, "rtsp")
+        assert (
+            self.api.surveillance_station.get_camera_live_view_path(1, "rtsp")
```

Comment:
I'm surprised you didn't set the base URL anywhere into the tests. It would be nice to have it to understand your use case.